sql_export_excel/tests/test_sql_query_excel.py

```diff
@@ -41,12 +41,7 @@ def test_excel_file_generation(self):
                 "sql_export_id": query.id,
             }
         )
-        workbook = self.get_workbook_from_query(wizard)
-        ws = workbook.active
-        # Check values, header should be here by default
-        self.assertEqual(ws.cell(row=1, column=1).value, "firstcol")
-        self.assertEqual(ws.cell(row=2, column=1).value, "testcol1")
```

Comment:
I don't see any reason to remove these lines from the test.
If it fails, it means there is a problem with the module